### PR TITLE
update readme with current info and badges

### PR DESCRIPTION
**Pull request by drochetti**
- added badges
- fixed code formatting/syntax
- updated category description

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,57 +43,69 @@ Amplify requires Xcode 11 or higher to build.
...
 ### CocoaPods
 
-1. Amplify for iOS is available through [CocoaPods](http://cocoapods.org). If you have not installed CocoaPods, install CocoaPods by running the command:
+| :information_source: For more detailed instructions, follow the getting started guides in our [documentation site](https://docs.amplify.aws/lib/q/platform/ios)   |
+|-------------------------------------------------|
 
-        $ gem install cocoapods
-        $ pod setup
+1. Amplify for iOS is available through [CocoaPods](http://cocoapods.org). If you have not installed CocoaPods, install CocoaPods by running the command:
+    ```
```
Comment (palpatim) [inline README.md:50]: nit here and throughout: add `bash` language specifier

File: README.md [hunk 2]
```diff
@@ -7,18 +7,23 @@ Our default implementation works with Amazon Web Services (AWS), but AWS Amplify
...
-- [**Predictions**](https://aws-amplify.github.io/docs/ios/predictions): Provides a solution for using AI and ML cloud services to enhance your application.
+- [Analytics](https://docs.amplify.aws/lib/analytics/getting-started/q/platform/ios) - for logging metrics and understanding your users.
+- [API (GraphQL)](https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/ios) - for adding a GraphQL endpoint to your app.
+- [API (REST)](https://docs.amplify.aws/lib/restapi/getting-started/q/platform/ios) - for adding a REST endpoint to your app.
+- [Authentication](https://docs.amplify.aws/lib/auth/getting-started/q/platform/ios) - for managing your users.
+- [DataStore](https://docs.amplify.aws/lib/datastore/getting-started/q/platform/ios) - for making it easier to program for a distributed data store for offline and online scenarios.
+- [Predictions](https://docs.amplify.aws/lib/predictions/getting-started/q/platform/ios) - to detect text, images, and more!
+- [Storage](https://docs.amplify.aws/lib/storage/getting-started/q/platform/ios) - store complex objects like pictures and videos to the cloud.
```
Comment (lawmicha) [inline README.md:22]: Can we align the descriptions with Android? ie. copy paste from Android https://github.com/aws-amplify/amplify-android/blob/main/README.md then we can work on the wording in this comment if you want to adjust it, then apply to both for alignment 

- **[Authentication](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android)**
  APIs and building blocks for developers who want to create user authentication
  experiences.
- **[Storage](https://docs.amplify.aws/lib/storage/getting-started/q/platform/android)**
  Provides a simple mechanism for managing user content for your app in public,
  protected or private storage buckets
- **[DataStore](https://docs.amplify.aws/lib/datastore/getting-started/q/platform/android)**
  Provides a programming model for leveraging shared and distributed data
  without writing additional code for offline and online scenarios.
- **[API
  (GraphQL)](https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/android)**
  Interact with your GraphQL server or AWS AppSync API with an easy-to-use &
  configured GraphQL client.
- **[API
  (REST)](https://docs.amplify.aws/lib/restapi/getting-started/q/platform/android)**
  Provides a simple solution when making HTTP requests. It provides an
  automatic, lightweight signing process which complies with AWS Signature
  Version 4.
- **[Analytics](https://docs.amplify.aws/lib/analytics/getting-started/q/platform/android)**
  Easily collect analytics data for your app. Analytics data includes user
  sessions and other custom events.
- **[Predictions](https://docs.amplify.aws/lib/predictions/getting-started/q/platform/android)**
  Connect your application with machine learning cloud services to enhance your
  application with natural language processing, computer vision, text to speech,
  and more.

Comment (lawmicha) [reply README.md:22]: or we can take the current suggestions in the PR and update Android's readme

Comment (drochetti) [reply README.md:22]: I'm using the content from our docsite, which is aligned between Android and iOS. I'm happy to use Android's, as long as the docsite is updated as well.

**Pull request merged**